.github/workflows/action-ros-ci-with-setup-ros.yml

```diff
@@ -0,0 +1,24 @@
+name: Build and test (actions-ros + setup-ros on ubuntu:focal)
+
+on: pull_request
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    container: ubuntu:focal
+
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+
+      - name: Setup ROS environment
+        uses: ros-tooling/setup-ros@v0.1
+        with:
+          required-ros-distributions: foxy
+
+      - name: Run action-ros-ci
+        uses: ros-tooling/action-ros-ci@v0.1
+        with:
+          package-name: obstacle_stop_planner_refine
+          target-ros2-distro: foxy
+          vcs-repo-file-url: build_depends.repos
```

Comment:
@KeisukeShima Should we keep the existing `build_and_test.yml` or will replace it with this?

Reply:
@kenji-miyake I'd like to replace old file with this.

Reply:
@KeisukeShima Then please remove it? :thinking:

Reply:
Done! ee668db

Reply:
@KeisukeShima By the way, the file name `action-ros-ci-with-setup-ros.yml` is okay for you? I feel renaming this to `build_and_test.yml` is better.

Reply:
@kenji-miyake It's better. Renaming c93c459